integration/js/app/video_test/video_test.html

```diff
@@ -62,6 +62,76 @@ <h2>Join a meeting</h2>
           Unbind Video Element
         </button>
       </form>
+
+       
+      <form id="add-video-tile-form">
+        <button id="add-video-tile-btn" type="submit">
+          addVideoTile
+        </button>
+        <label for="add-video-tile-label">Tile Id: </label>
+        <span id="video-tile-id"></span>
+      </form>
+
+      <form id="has-started-local-video-tile-form">
+        <button id="has-started-local-video-tile-btn" type="submit">
+          hasStartedLocalVideoTile
+        </button>
+        <label for="has-started-local-video-tile-label">Value: </label>
+        <span id="has-started-local-video-tile-boolean"></span>
+      </form>
+
+      <form id="get-local-video-tile-form">
+        <button id="get-local-video-tile-btn" type="submit">
+          getLocalVideoTile
+        </button>
+        <label for="local-video-tile-label">Tile Id: </label>
+        <span id="local-video-tile-id"></span>
+      </form>
+
+      <form id="get-all-video-tiles-form">
+        <button id="get-all-video-tiles-btn" type="submit">
+          getAllVideoTiles
+        </button>
+        <label for="total-video-tiles-label">Total Tiles Count: </label>
+        <span id="total-video-tiles-count"></span>
+      </form>
+
+      <form id="get-all-remote-video-tiles-form">
+        <button id="get-all-remote-video-tiles-btn" type="submit">
+          getAllRemoteVideoTiles
+        </button>
+        <label for="total-remote-video-tiles-label">Total Remote Tiles Count: </label>
+        <span id="total-remote-video-tiles-count"></span>
+      </form>
+
+      <form id="have-video-tile-for-attendeeId-form">
+        <button id="have-video-tile-for-attendeeId-btn" type="submit">
+          haveVideoTileForAttendeeId
+        </button>
+        <label for="have-video-tile-for-attendeeId-label">Value: </label>
+        <span id="have-video-tile-for-attendeeId-boolean"></span>
+      </form>
+
+      <form id="have-video-tiles-with-streams-form">
+        <button id="have-video-tiles-with-streams-btn" type="submit">
+          haveVideoTilesWithStreams
+        </button>
+        <label for="have-video-tiles-with-streams-label">Value: </label>
+        <span id="have-video-tiles-with-streams-boolean"></span>
+      </form>
+
+      <form id="send-tile-state-update-form">
+        <button id="send-tile-state-update-btn" type="submit">
+          sendTileStateUpdate
```

Comment:
If we do include this API, I think we may want to be able to pass in a TileState parameter